lib/generators/boring/database_cleaner/active_record/rspec/install/install_generator.rb

```diff
@@ -17,54 +36,21 @@ def add_database_cleaner_active_record_gem
             end
           end
 
-          def configure_database_cleaner
-            verify_presence_of_rspec_gem
-
-            create_file 'spec/support/database_cleaner.rb' do
-              rspec_config
-            end
-            inject_into_file 'spec/rails_helper.rb',
-                             "require 'spec/support/database_cleaner.rb'\n\n",
-                             before: /\A/
-          end
-
-          private
-
-          def rspec_config
-            <<~RUBY
-              RSpec.configure do |config|
-                config.before(:suite) do
-                  DatabaseCleaner.clean_with(:truncation)
-                end
-
-                config.before do
-                  DatabaseCleaner.strategy = :transaction
-                end
-
-                config.before(:each, js: true) do
-                  DatabaseCleaner.strategy = :truncation
-                end
-
-                config.before do
-                  DatabaseCleaner.start
-                end
-
-                config.after do
-                  DatabaseCleaner.clean
-                end
-              end
-            RUBY
+          def disable_transactional_fixtures!
+            gsub_file "spec/rails_helper.rb",
```

Comment:
Let's do this inside configure_webmock as well

Also shouldn't this option be false only when capybara (or system test) is also used? By default this option is true for a good reason; clean database without any records for every example.

Reply:
I don't understand this "Let's do this inside configure_webmock as well"

when we use database cleaner we can disable transactional fixtures as 
```config.before do
    DatabaseCleaner.strategy = :transaction
  end```
  do same thing